diff --git a/examples/platforms/efr32mg12/openthread-core-efr32-config.h b/examples/platforms/efr32mg12/openthread-core-efr32-config.h
index d1cbde654..09d31ebf4 100644
--- a/examples/platforms/efr32mg12/openthread-core-efr32-config.h
+++ b/examples/platforms/efr32mg12/openthread-core-efr32-config.h
@@ -38,15 +38,6 @@
 #ifndef OPENTHREAD_CORE_EFR32_CONFIG_H_
 #define OPENTHREAD_CORE_EFR32_CONFIG_H_
 
-/**
- * @def OPENTHREAD_CONFIG_LOG_OUTPUT
- *
- * The efr32 platform provides an otPlatLog() function.
- */
-#ifndef OPENTHREAD_CONFIG_LOG_OUTPUT /* allow command line override */
-#define OPENTHREAD_CONFIG_LOG_OUTPUT OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED
-#endif
-
 /*
  * @def OPENTHREAD_CONFIG_RADIO_915MHZ_OQPSK_SUPPORT
  *
diff --git a/examples/platforms/efr32mg12/radio.c b/examples/platforms/efr32mg12/radio.c
index 764be7183..5e0596a20 100644
--- a/examples/platforms/efr32mg12/radio.c
+++ b/examples/platforms/efr32mg12/radio.c
@@ -34,8 +34,10 @@
 
 #include <assert.h>
 
-#include "openthread-system.h"
+#include "openthread-core-efr32-config.h"
 #include <openthread/config.h>
+
+#include "openthread-system.h"
 #include <openthread/platform/alarm-milli.h>
 #include <openthread/platform/diag.h>
 #include <openthread/platform/radio.h>
@@ -50,7 +52,6 @@
 #include "em_core.h"
 #include "em_system.h"
 #include "hal-config.h"
-#include "openthread-core-efr32-config.h"
 #include "pa_conversions_efr32.h"
 #include "platform-band.h"
 #include "rail.h"
diff --git a/examples/platforms/efr32mg12/system.c b/examples/platforms/efr32mg12/system.c
index 7b1aa3c25..11dae8700 100644
--- a/examples/platforms/efr32mg12/system.c
+++ b/examples/platforms/efr32mg12/system.c
@@ -35,6 +35,9 @@
 #include <assert.h>
 #include <string.h>
 
+#include "openthread-core-efr32-config.h"
+#include <openthread/config.h>
+
 #include "openthread-system.h"
 #include <openthread/platform/uart.h>
 
@@ -52,7 +55,6 @@
 #include "sl_mpu.h"
 #include "sl_sleeptimer.h"
 
-#include "openthread-core-efr32-config.h"
 #include "platform-efr32.h"
 
 #if (HAL_FEM_ENABLE)
